docs/scos/dev/glue-api-guides/202311.0/decoupled-glue-api.md

```diff
@@ -1,36 +1,36 @@
 ---
 title: Decoupled Glue API
 description: Learn about the process of handling API requests through GlueStorefront and GlueBackoffice layers.
-last_updated: Jul 11, 2023
+last_updated: November 29, 2023
 template: glue-api-storefront-guide-template
 ---
 
-The Spryker Decoupled Glue API is a set of a few API applications like *Glue Storefront API (SAPI)* and *Glue Backend API (BAPI)* of the Spryker Commerce OS. Those applications are built to be used as a contract for accessing Storefront or Backoffice functionality through API. Those applications know how to read and interpret API resources and leverage feature modules that expose existing Spryker functionality.
+The Spryker Decoupled Glue API is a set of a few API applications like *Glue Storefront API* and *Glue Backend API* of the Spryker Commerce OS. Those applications are built to be used as a contract for accessing Storefront or Backoffice functionality through API. Those applications know how to read and interpret API resources and leverage feature modules that expose existing Spryker functionality.
 
 ## Existing Glue Applications
 
 Out of the box, Spryker Commerce OS provides three API applications:
-* Old Glue API application that can be used as a fallback.
-* New Glue Storefront API (SAPI) that is a replacement for the old Glue and can be used for the same purpose.
-* Glue Backend API (BAPI) that can be used to provide API access for the Backoffice functionality directly without any additional RPC calls.
+* Legacy Glue API application that can be used as a fallback.
```

Comment:
"...can be used as a fallback." Why? in which cases? please provide context and scenarios. And how can I use it as fallback?